### PR TITLE
Edit labels - Error Messages on client validation

### PR DESCRIPTION
**Pull request by ivoalmeida**
Implement validation on key stroke  and display warning message to warn users of invalid input.

https://issues.redhat.com/browse/RHSTOR-4119

### CODE REVIEW

File: packages/shared/generic/FormAlertInline.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { FormAlert, Alert } from '@patternfly/react-core';
+
+export type FormAlertInlineProps = {
+  title: string;
+  message: string;
+  variant?: 'success' | 'danger' | 'warning' | 'info' | 'default';
```
Comment (bipuladh) [inline packages/shared/generic/FormAlertInline.tsx:7]:
```suggestion
  variant?: AlertProps['variant'];
```

File: packages/shared/modals/EditAnnotations.tsx [hunk 2]
```diff
@@ -116,7 +116,14 @@ export const AnnotationsModal: React.FC<AnnotationsModalProps> = ({
...
         />
-        <div>{errorMessage && <ErrorMessage message={errorMessage} />}</div>
+        <div>
+          {errorMessage && (
+            <FormAlertInline
+              title={t('An error occurred')}
+              message={errorMessage}
+            />
+          )}
+        </div>
```
[the remark this answers is not in the document]
Comment (ivoalmeida) [reply packages/shared/modals/EditAnnotations.tsx]: done 👍🏽

File: packages/shared/generic/FormAlertInline.tsx (hunk 1 above)
Comment (ivoalmeida) [reply packages/shared/generic/FormAlertInline.tsx]: done 👍🏽

**Comment by SanjalKatiyar** [comment]
we follow one commit per PR (mostly), we can squash the commits of this PR into one or two (at max) @ivoalmeida

**Comment by SanjalKatiyar** [comment]
please run "yarn i18n" command as well on your patch, it is needed for translations (whenever we make any changes to         "t( )" function)...

**Comment by bipuladh** [comment]
Screenshot:
image-2023-01-11-10-25-40-347

**Review by bipuladh — APPROVED** [review]
/lgtm

**Pull request merged**